src/condor_utils/file_transfer.cpp

```diff
@@ -3840,15 +3841,17 @@ FileTransfer::DoUpload(filesize_t *total_bytes_ptr, ReliSock *s)
 	jobAd.LookupBool( ATTR_PRESERVE_RELATIVE_PATHS, preserveRelativePaths );
 
 	FileTransferList filelist;
+	dprintf( D_KEYBOARD, ">>> DoUpload(), before ExpandFileTransferList(): InputFiles = %s\n", FilesToSend->to_string().c_str() );
 	ExpandFileTransferList( FilesToSend, filelist, preserveRelativePaths );
+	for( auto & i: filelist ) { dprintf( D_KEYBOARD, ">>> DoUpload(), file-item after ExpandFileTransferList(): %s -> %s\n", i.srcName().c_str(), i.destDir().c_str() ); }
 
 	// Remove any files from the catalog that are in the ExceptionList
 	if (ExceptionFiles) {
-		auto enditer = 
+		auto enditer =
```

Comment:
Please avoid extra whitespace movement here as it will conflict with #258's merge.

Reply:
Fair.  Done.  Will be pushed shortly.